code/datalab-app/src/api/infrastructure/__snapshots__/stackApi.spec.js.snap

```diff
@@ -0,0 +1,10 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Stack API create stack should handle api errors 1`] = `
+Object {
+  "name": "notebookName",
```

Comment:
This is still has notebookName in the errors?

Reply:
Checked and correct as this is echoing the request passed in.